python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -55,6 +68,50 @@ def __init__(self, database, driver='sqlite', host=None, port=None):
         self.host = host
         self.port = port
         self.database = database
+        self._seeing_column = 'FWHMeff'
+
+        # a dict keyed on the user interface (i.e. args to getObservationMetaData)
+        # names of OpSim data columns.  Returns a tuple that is the
+        # (name of data in OpSim, transformation to go from user interface to OpSim units, dtyp in OpSim)
+        self._user_interface_to_opsim = {'obsHistID': ('obsHistID', None, np.int64),
```

Comment:
OK. I was more worried about being able to understand what is going on. But, I agree with keeping this as a protected variable ... maybe having some more documentation about this is a better way to address that  than trying to make this public.